deploy/eck-operator/templates/webhook.yaml

```diff
@@ -12,7 +12,9 @@ metadata:
 {{- end }}
 webhooks:
 - clientConfig:
+    {{- if not .Values.webhook.manageCerts }}
```

Comment:
https://github.com/elastic/cloud-on-k8s/blob/bebf07b3959e4475d9d051c0873190575ff04a33/deploy/eck-operator/values.yaml#L106

Should we also include the check to ensure that `certManagerCert` is `null`?

Reply:
Absolutely.